components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Settings.php

```diff
@@ -265,6 +238,11 @@ public function options( $options, $pod ) {
 					$options['pods-pfat'][ $k ] = array_merge( $option, $pick );
 				}
 			}
+
+			// remove single from taxonomy
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: ae422a7d3e98aa43a8478678dc89b3c1</sub>